src/common/base/abstractAsyncOperation.ts

```diff
@@ -98,12 +98,23 @@ export abstract class AsyncCommand extends SfCommand<AsyncOperationResultJson> {
    * @param wait
    */
   private async startMonitoring(wait: Duration): Promise<void> {
-    const streamer: DoceMonitor = getAsyncOperationStreamer(
-      this.targetOrg,
-      wait,
-      this.asyncOperationId,
-      this.outputService
+    const asyncJob: AsyncOperationResult = await fetchAsyncOperationResult(
+      this.targetOrg.getConnection(),
+      this.asyncOperationId
     );
-    await streamer.monitor();
+    if (
+      asyncJob.sf_devops__Status__c === undefined ||
```

Comment:
In which case sf_devops__Status__c could be undefined? The field is a required picklist so should always have value right?